### PR TITLE
大改staff名单

### PR DESCRIPTION
**Pull request by C29H25N3O5**
Redesigned the layout of the staff roll.

### CODE REVIEW

File: parts/language/lang_id.lua [hunk 1]
```diff
@@ -211,19 +211,98 @@ return{
...
+        "Diprogram, Dikembangkan, Dan Dirancang Oleh",
+        "MrZ",
+        "",
+        "Musik Dibuat Menggunakan",
+        "Beepbox",
+        "FL Studio",
+        "FL Mobile",
+        "Logic Pro X",
+        "",
+        "[POWERED BY LÖVE]",
```
[the remark this answers is not in the document]
Comment (C29H25N3O5) [reply parts/language/lang_id.lua:226]: In fact all languages uses “POWERED BY LÖVE”
MrZ said that he would later replace this by a stylized logo of LÖVE from the official website, so leave it intact please

**Comment by C29H25N3O5** [comment]
> I found a couple of translation errors on the Indonesian language

All clear.

**Comment by Gompyn** [comment]
ImpleLee and Gompyn are all me.
List me as Gompyn.

**Comment by C29H25N3O5** [comment]
> ImpleLee and Gompyn are all me.
> List me as Gompyn.

Clear.

**Pull request merged**